src/session.ts

```diff
@@ -188,6 +199,135 @@ export function valueSession(): void {
   Session.current().toFile("test.json");
 }
 
+let marginalSession: Session | null = null;
+let marginalSessionDiff: Session | null = null;
+let barfSession: Session | null = null;
+let barfSessionStartTurns = totalTurnsPlayed();
+let continueMeatTracking = true;
+let numTrackedItemTurns: number | null = null;
+export function trackBarfSessionStatistics(): void {
+  // If we are overdrunk, don't track statistics
+  if (myInebriety() > inebrietyLimit()) return;
+
+  // Start barfSession if we have not done so
+  if (!barfSession) {
+    barfSession = Session.current();
+    barfSessionStartTurns = totalTurnsPlayed();
+  }
+
+  // Start tracking items if at least one of these is true
+  // 1) We have run at least 100 barf turns
+  // 2) We have less than 200 adv to run
+  if (
+    !marginalSession &&
```

Comment:
i don't mind these two as conditions for item tracking, but if they are going to be our only major conditions, we need a way somewhere in here to cut marginal reporting entirely if the user is running <200 turns overall -- otherwise it'll contain free fights and it will be a bad overestimate.

Reply:
This is only triggered during barfTurns() so it should never consider free fights